datafusion/sqllogictest/test_files/repartition_scan.slt

```diff
@@ -185,12 +185,12 @@ COPY  (VALUES (1), (2), (3), (4), (5)) TO 'test_files/scratch/repartition_scan/j
 (FORMAT json, SINGLE_FILE_OUTPUT true);
 
 statement ok
-CREATE EXTERNAL TABLE json_table(column1 int)
+CREATE EXTERNAL TABLE json_table (column1 int)
 STORED AS json
 LOCATION 'test_files/scratch/repartition_scan/json_table/';
 
 query I
-select * from json_table;
+select * from "json_table";
```

Comment:
This is needed because JSON_TABLE is now parsed specially in SQLParser: https://github.com/sqlparser-rs/sqlparser-rs/pull/1062